test/fixtures/08-tasks/task-handlers.ts

```diff
@@ -0,0 +1,60 @@
+import type { FromSchema } from 'json-schema-to-ts';
+import { sbvrUtils } from '../../../src/server-glue/module';
+
+// Define JSON schema for accepted parameters
+const createDeviceParamsSchema = {
```

Comment:
Note: I also investigated using [zod](https://www.npmjs.com/package/zod) to define parameter object shapes, but decided to stick with JSON Schema as its a commonly used standard. Consumers can still define their shapes with zod and then convert to JSON Schema using something like [zod-to-json-schema](https://www.npmjs.com/package/json-schema-to-zod) when defining task handlers.

Reply:
FWIW I have used zod for quite a few things and altough I like the library a lot I think json sechema better serves in here as zod is a bulkier library that can do much more, and we would probably want to leverage all the zod type validation on several more places